core/gethwrappers/generated/ocr2dr_oracle/ocr2dr_oracle.go

```diff
@@ -170,6 +177,28 @@ func (_OCR2DROracle *OCR2DROracleTransactorRaw) Transact(opts *bind.TransactOpts
 	return _OCR2DROracle.Contract.contract.Transact(opts, method, params...)
 }
 
+func (_OCR2DROracle *OCR2DROracleCaller) EstimateCost(opts *bind.CallOpts, data []byte, billing OCR2DRRegistryInterfaceRequestBilling) (*big.Int, error) {
+	var out []interface{}
+	err := _OCR2DROracle.contract.Call(opts, &out, "estimateCost", data, billing)
```

Comment:
why doesn't this conform to the Go pattern seen elsewhere which would be
`valueReturnedByFunc, err := _OCR2DROracle.contract.Call(opts, &out, "estimateCost", data, billing)`

Reply:
Not sure, this is auto-generated.